stable/jaeger-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.9.0
-appVersion: 1.13.1
+version: 2.10.0
+appVersion: 1.14.0
```

Comment:
can you update the values in the readme to match? TIA

Reply:
Absolutely. Sorry for not doing that earlier.